### PR TITLE
adding postgrest evn variable docs

### PR DESCRIPTION
**Pull request by adishM98**
added postgrest evn variables docs with explanation

### CODE REVIEW

File: docs/docs/setup/env-vars.md [hunk 1]
```diff
@@ -250,3 +250,29 @@ This can be an absolute path, or relative to main HTML file.
...
+:::info Please make sure that DB_URI is given in the format `postgrest://[PG_USER]:[PG_PASS]@[PG_HOST]/[TOOLJET_DB]` :::
+
+PostgRest database configuration (required)
+
+| variable           | description            |
+| ------------------ | ---------------------- |
+| PGRST_HOST         | postgrest database host|
+| PGRST_JWT_SECRET   | JWT token client provided for authentication |
+| PGRST_DB_URI       | database connection string |
+| PGRST_LOG_LEVEL    | Default value is `info` |
```
Comment (akshaysasidrn) [inline docs/docs/setup/env-vars.md:276]: We can only suggest using `info` here as its value by default unless the user wants to set it as something else.

We can also add a link to postgrest configuration docs below.

File: docs/docs/setup/env-vars.md [hunk 2]
```diff
@@ -250,3 +250,29 @@ This can be an absolute path, or relative to main HTML file.
...
+
+:::tip If you have openssl installed, you can run the following command `openssl rand -hex 32` to generate the value for `PGRST_JWT_SECRET`. 
+
+If this parameter is not specified then PostgREST refuses authentication requests. 
+:::
+
+
+:::info Please make sure that DB_URI is given in the format `postgrest://[PG_USER]:[PG_PASS]@[PG_HOST]/[TOOLJET_DB]` :::
+
+PostgRest database configuration (required)
```
Comment (akshaysasidrn) [inline docs/docs/setup/env-vars.md:269]: PostgREST web server configuration*

File: docs/docs/setup/env-vars.md [hunk 3]
```diff
@@ -250,3 +250,29 @@ This can be an absolute path, or relative to main HTML file.
...
+
+| variable           | description            |
+| ------------------ | ---------------------- |
+| PGRST_HOST         | postgrest database host|
+| PGRST_JWT_SECRET   | JWT token client provided for authentication |
+| PGRST_DB_URI       | database connection string |
+| PGRST_LOG_LEVEL    | Default value is `info` |
+| TOOLJET_DB         | Default value is `tooljet_db` |
```
Comment (akshaysasidrn) [inline docs/docs/setup/env-vars.md:277]: This config will be for tooljet server

File: docs/docs/setup/env-vars.md (hunk 1 above)
Comment (adishM98) [reply docs/docs/setup/env-vars.md]: added the above mentioned changes

File: docs/docs/setup/env-vars.md (hunk 2 above)
Comment (adishM98) [reply docs/docs/setup/env-vars.md]: changed the heading to PostgRest database configuration to PostgREST web server configuration.

File: docs/docs/setup/env-vars.md (hunk 3 above)
Comment (adishM98) [reply docs/docs/setup/env-vars.md]: added tooljet db under tooljet server heading

File: docs/versioned_docs/version-2.0.0-beta/setup/env-vars.md [hunk 4]
```diff
@@ -15,6 +15,14 @@ Both the ToolJet server and client requires some environment variables to start
 | ------------ | --------------------------------------------------------------- |
 | TOOLJET_HOST | the public URL of ToolJet client ( eg: https://app.tooljet.com )  |
 
+
+#### Tooljet Database (optional)
```
Comment (akshaysasidrn) [inline docs/versioned_docs/version-2.0.0-beta/setup/env-vars.md:19]: Need to also add:

`ENABLE_TOOLJET_DB`
`PGRST_HOST`
`PGRST_JWT_SECRET`

**Review by akshaysasidrn — CHANGES_REQUESTED** [review]

File: docs/versioned_docs/version-2.0.0-beta/setup/env-vars.md [hunk 5]
```diff
@@ -250,3 +258,28 @@ This can be an absolute path, or relative to main HTML file.
...
+
+:::info Please make sure that DB_URI is given in the format `postgrest://[PG_USER]:[PG_PASS]@[PG_HOST]/[TOOLJET_DB]` :::
+
+PostgREST web server configuration (required)
+
+| variable           | description            |
+| ------------------ | ---------------------- |
+| PGRST_HOST         | postgrest database host|
+| PGRST_JWT_SECRET   | JWT token client provided for authentication |
+| PGRST_DB_URI       | database connection string |
```
[the remark this answers is not in the document]
Comment (adishM98) [reply docs/versioned_docs/version-2.0.0-beta/setup/env-vars.md]: changed | PGRST_DB_URI       | database connection string |  to | PGRST_DB_URI | database connection string for tooljet database |

File: docs/versioned_docs/version-2.0.0-beta/setup/env-vars.md [hunk 6]
```diff
@@ -250,3 +258,28 @@ This can be an absolute path, or relative to main HTML file.
...
+| PGRST_LOG_LEVEL    | `n/a` or `info`. Default value is `info` |
```
[the remark this answers is not in the document]
Comment (adishM98) [reply docs/versioned_docs/version-2.0.0-beta/setup/env-vars.md]: removed default value

File: docs/versioned_docs/version-2.0.0-beta/setup/env-vars.md [hunk 7]
```diff
@@ -250,3 +258,28 @@ This can be an absolute path, or relative to main HTML file.
...
+
+:::tip If you have openssl installed, you can run the following command `openssl rand -hex 32` to generate the value for `PGRST_JWT_SECRET`. 
+
+If this parameter is not specified then PostgREST refuses authentication requests. 
+:::
+
+
+:::info Please make sure that DB_URI is given in the format `postgrest://[PG_USER]:[PG_PASS]@[PG_HOST]/[TOOLJET_DB]` :::
+
+PostgREST web server configuration (required)
```
[the remark this answers is not in the document]
Comment (adishM98) [reply docs/versioned_docs/version-2.0.0-beta/setup/env-vars.md]: made PostgREST web server configuration as a heading

File: docs/versioned_docs/version-2.0.0-beta/setup/env-vars.md (hunk 4 above)
Comment (adishM98) [reply docs/versioned_docs/version-2.0.0-beta/setup/env-vars.md:19]: added:

ENABLE_TOOLJET_DB
PGRST_HOST
PGRST_JWT_SECRET

**Review by akshaysasidrn — APPROVED** [review]

**Pull request merged**